README.md

```diff
@@ -0,0 +1,13 @@
+# nodecg-io-docs
+
+[![Feature Requests](https://img.shields.io/github/issues/codeoverflow-org/nodecg-io-docs/enhancement?label=Feature%20Requests&style=flat-square)](https://github.com/codeoverflow-org/nodecg-io/labels/enhancement)
+[![Bugs](https://img.shields.io/github/issues/codeoverflow-org/nodecg-io-docs/bug?label=Bugs&style=flat-square)](https://github.com/codeoverflow-org/nodecg-io/labels/bug)
+[![Pull Requests](https://img.shields.io/github/issues-pr/codeoverflow-org/nodecg-io-docs?label=Pull%20Requests&style=flat-square)](https://github.com/codeoverflow-org/nodecg-io/pulls)
+[![License](https://img.shields.io/github/license/codeoverflow-org/nodecg-io-docs?label=License&style=flat-square)](https://github.com/codeoverflow-org/nodecg-io/blob/master/LICENSE)
+
+## About nodecg-io-docs
+
+This is the documentation repository for [nodecg-io](https://github.com/codeoverflow-org/nodecg-io), a NodeCG-bundle which will implement Social Media API's in the NodeCG framework. For more info see the main repo, or visit the hosted documentation on [nodecg.io](https://nodecg.io/). 
+
+## How to contribute
+If you want to contribute to the documentation you can create an [issue](https://github.com/codeoverflow-org/nodecg-io-docs/issues). Before contributing had over to the [How to contribute](https://nodecg.io/contribute/) - Guide.
```

Comment:
typo: `had` over.
Issues on this repo are still deactivated. Issues shouldn't be used for contributing to the docs, 

A contributing guide for the docs should encourage creating pull requests, show how to use the editt feature of mkdocs to directly edit a page and contain some information on how to serve the docs if the docs repo was cloned locally. 

Also https://nodecg.io/contribute is about contributing code, not docs.